api/nodes/views.py

```diff
@@ -566,3 +566,29 @@ def get_queryset(self):
             if addon.config.has_hgrid_files
             and addon.complete
         ]
+
+
+class NodeLogList(generics.ListAPIView, NodeMixin):
+    """ Recent Log Activity
+
+    This allows users to be able to get log information. This will allow more interesting
```

Comment:
This should be brought up to date with our more robust api documentation standards.